shared/output/cloud/speech_v1/lib/google/cloud/speech/v1/speech/operations.rb

```diff
@@ -324,11 +324,11 @@ def delete_operation request, options = nil
             #     makes a best effort to cancel the operation, but success is not
             #     guaranteed.  If the server doesn't support this method, it returns
             #     `google.rpc.Code.UNIMPLEMENTED`.  Clients can use
-            #     [Operations.GetOperation][google.longrunning.Operations.GetOperation] or
+            #     `Operations.GetOperation` or
             #     other methods to check whether the cancellation succeeded or whether the
             #     operation completed despite cancellation. On successful cancellation,
             #     the operation is not deleted; instead, it becomes an operation with
-            #     an [Operation.error][google.longrunning.Operation.error] value with a [google.rpc.Status.code][google.rpc.Status.code] of 1,
+            #     an {Google::Longrunning::Operation#error Operation.error} value with a {Google::Rpc::Status#code google.rpc.Status.code} of 1,
```

Comment:
Wasn't the intent in the code to remove the Google::Longrunning?

Reply:
The intent is to omit specifically the _service_ `Google::Longrunning::Operations` (plural), but not anything else under `Google::Longrunning` including in this case, the _message_ `Google::Longrunning::Operation` (singular).

Reply:
well, this really needs to be a comment in `transform_xref`